charts/member-agent/values.yaml

```diff
@@ -35,6 +35,17 @@ config:
   identityKey: "identity-key-path"
   identityCert: "identity-cert-path"
   CABundle: "ca-bundle-path"
+  cloudConfig:
+    cloud: ""
+    tenantId: ""
+    subscriptionId: ""
+    useManagedIdentityExtension: false
+    userAgent: ""
```

Comment:
We also need userAssignedIdentityID, aadClientId, aadClientSecret, and resourceGroup 4 fields